doc/make/wsl.md

```diff
@@ -38,17 +37,20 @@ with a functioning infoview, syntax coloring and tooltips.
 
 **Connection to server is erroring. Shutting down server.**
 
-Check that your `leanpkg` was created using the WSL linux version of
-`leanpkg` and not with the windows version of the tool.
+If your `leanpkg` has a `build` folder try deleting it.  Lean .olean
+binaries are not cross platform so if the binaries were copied from a
+different platform they need to be deleted so that the new build is
+created for your platform.

```

Comment:
Do you mean something other than `.olean`? I just downloaded the Linux and Windows nightlies and they are identical according to `diff`.

Reply:
weird, someone told me somewhere that .olean files were not cross platform.  I'm glad to hear they are.  I'll double check this issue I was running into and report more details...